backend/app/api/journal.py

```
@@ -105,6 +116,197 @@ async def parse_stt(
+        # 모듈 로드 자체 실패 — 환경/배포 이슈일 수 있어 운영자가 인지하도록 warning.
+        logger.warning(
+            "journal.parse_photos.pesticide_matcher_unavailable err=%s", exc
+        )
+
+    # 사진 디스크 저장 + DB 행 생성 (entry_id=null 임시 사진)
+    # strict=True: images/files 는 위에서 1:1 로 만들어졌으므로 항상 동일 길이 — 깨지면 즉시 실패.
+    saved_ids: list[int] = []
+    for img_bytes, f in zip(images, files, strict=True):
+        try:
+            meta = save_photo(current_user.id, img_bytes)
+        except Exception as save_err:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)